### PR TITLE
[TFF-217] Style main index page

### PR DESCRIPTION
**Pull request by mattmahn**
### Summary

I made the main/index page more prettier. I included a font, so don't mind `assets/fonts/`.
The decrease in coverage is due to the lost item functionality being moved to the index page. There are no tests for it because there weren't any tests for it before.

### Checklist

- [x] Have you added unit and functional tests as appropriate?
- [x] Did you update/add documentation for new methods or changed functionality?
- [x] Have you merged the target branch down into this one?

### How to Review

1. Start the app
1. Admire the index page
1. Test the navigation to make I didn't break/forget anything
1. Verify the code is adequate

### Links

- [TFF-217](https://msoese.atlassian.net/browse/TFF-217)

### CODE REVIEW

File: src/views/components/input-modal.jsx [hunk 1]
```diff
@@ -32,7 +32,7 @@ export default class InputModal extends React.Component {
 
     renderWarning(){
         if(this.state.invalid === undefined ? this.state.invalid : this.props.invalid){
-            return <span className="invalidText">{this.props.errorMessage === undefined ? 'Please only use Alphanumeric characters.' : this.props.errorMessage}<br/></span>;
+            return <span className="invalidText">{this.props.errorMessage === undefined ? 'Please only use alspanhanumeric characters.' : this.props.errorMessage}</span>;
```
Comment (tondef001) [inline src/views/components/input-modal.jsx]: It appears something went wrong here...

File: src/views/pages/index.jsx [hunk 2]
```diff
@@ -1,18 +1,118 @@
...
     render() {
         return (
             <div id='index'>
+                <InputModal
+                    message='Please enter the item or model address located under the barcode.'
+                    active={this.state.lostItemModalActive}
+                    onAccept={this.acceptItemLostAddr.bind(this)}
+                    onCancel={this.cancelItemLost.bind(this)}
+                    update={this.update.bind(this)}
+                    errorMessage='Equipment address is incorrect. Please double check adress!'
```
Comment (gearedmonkey) [inline src/views/pages/index.jsx]: I spelled address wrong, can you fix?

File: src/views/pages/index.jsx [hunk 3]
```diff
@@ -1,18 +1,118 @@
...
+                    acceptDisabled={this.state.inputModalAcceptDisabled}
+                />
                 <div id='links'>
-                  <button id='view-models' onClick={IndexController.getModels}>View all models</button><br/>
-                  <button id='view-items' onClick={IndexController.getItems}>View all items</button><br/>
-                  <button onClick={IndexController.getOverdueItems}>Overdue Items</button><br/>
-                  <button id='view-students' onClick={IndexController.getStudents}>View All Students</button><br/>
-                  <Link to='/students/upload'>Upload Student List</Link>
+                    <div id='view-models' onClick={IndexController.getModels}>
+                        <span>
```
Comment (gearedmonkey) [inline src/views/pages/index.jsx]: Why do you have the span tags wrapping each of the clickables? They don't seem to be styled in anyway.

File: src/views/pages/index.jsx [hunk 4]
```diff
@@ -1,18 +1,118 @@
...
+                            Models
+                        </span>
+                    </div>
+                    <div id='view-items' onClick={IndexController.getItems}>
+                        <span>
+                            <img src="../assets/images/list.svg"/><br/>
+                            Items
+                        </span>
+                    </div>
+                    <div id='new-model' onClick={() => hashHistory.push(`/models/new`)}>
```
Comment (gearedmonkey) [inline src/views/pages/index.jsx]: Can we keep the onClick's consistent and have them all use the IndexController? I know OmnibarController has a `leavePage(path)` function.

File: src/views/pages/index.jsx (hunk 2 above)
Comment (tondef001) [inline src/views/pages/index.jsx]: Fixed in the modal PR.

File: src/views/pages/index.jsx (hunk 3 above)
Comment (tondef001) [inline src/views/pages/index.jsx]: That's how I got the icons to be not off-center and the text to go all wonky.  Don't ask me, it just works.

File: src/views/pages/index.jsx (hunk 4 above)
Comment (tondef001) [inline src/views/pages/index.jsx]: agreed

**Pull request merged**